src/containerapp/azext_containerapp/tests/latest/test_containerapp_dotnet_component.py

```diff
@@ -0,0 +1,103 @@
+        dotnet_component_name = self.create_random_name(prefix='dotnet-aca', length=24)
+        location = "westus2"
+
+        env_create_cmd = f'containerapp env create -g {resource_group} -n {env_name} --location {location} --logs-destination none --enable-workload-profiles'
+        self.cmd(env_create_cmd)
+
+        # List DotNet Components
+        dotnet_component_list = self.cmd("containerapp env dotnet-component list -g {} --environment {}".format(resource_group, env_name)).get_output_in_json()
+        self.assertTrue(len(dotnet_component_list) == 0)
+
+        # Create DotNet Component
+        self.cmd('containerapp env dotnet-component create -g {} -n {} --environment {}'.format(resource_group, dotnet_component_name, env_name))
+
+        # Show DotNet Component
+        self.cmd('containerapp env dotnet-component show -g {} -n {} --environment {}'.format(resource_group, dotnet_component_name, env_name), checks=[
+            JMESPathCheck('name', dotnet_component_name),
+            JMESPathCheck('properties.componentType', "AspireDashboard"),
+            JMESPathCheck('properties.provisioningState', "Succeeded")
+        ])
+
+        # List DotNet Components
+        dotnet_component_list = self.cmd("containerapp env dotnet-component list -g {} --environment {}".format(resource_group, env_name)).get_output_in_json()
+        self.assertTrue(len(dotnet_component_list) == 1)
+
+        # Delete DotNet Component
+        self.cmd('containerapp env dotnet-component delete -g {} -n {} --environment {} --yes'.format(resource_group, dotnet_component_name, env_name), expect_failure=False)
+
+        # List DotNet Components
+        dotnet_component_list = self.cmd("containerapp env dotnet-component list -g {} --environment {}".format(resource_group, env_name)).get_output_in_json()
+        self.assertTrue(len(dotnet_component_list) == 0)
+
+    @AllowLargeResponse(8192)
+    @ResourceGroupPreparer(location="westus2")
+    def test_containerapp_dotnet_component_create_with_unsupported_component_type(self, resource_group):
+        env_name = self.create_random_name(prefix='aca-dotnet-env', length=24)
+        dotnet_component_name = self.create_random_name(prefix='dotnet-aca', length=24)
+        location = "westus2"
+
+        env_create_cmd = f'containerapp env create -g {resource_group} -n {env_name} --location {location} --logs-destination none --enable-workload-profiles'
+        self.cmd(env_create_cmd)
+
+        # List DotNet Components
+        dotnet_component_list = self.cmd("containerapp env dotnet-component list -g {} --environment {}".format(resource_group, env_name)).get_output_in_json()
+        self.assertTrue(len(dotnet_component_list) == 0)
+
+        # Create DotNet Component
+        self.cmd('containerapp env dotnet-component create -g {} -n {} --environment {} --type {}'.format(resource_group, dotnet_component_name, env_name, "test-component-type"), expect_failure=True)
```

Comment:
In CLI it will throw SystemExit 2 for `Parser error; check input to command line`.
```
with self.assertRaisesRegex(SystemExit, '2'):
    self.cmd('containerapp env dotnet-component create -g a -n b --environment c --type {}'.format("test-component-type"))
```

Reply:
Thanks!